### DISCUSSION TITLE
Moved Resources from one subscription to another one

### DISCUSSION THREAD

**Discussion by sainathyv** [microsoft/finops-toolkit#1002] [Q&A] [answered]
Hi All,
Moved all AOE Resources from one subscription to new one and updated variables in Automation Account. 
My All jobs are failling with the below error. Don't know from where it is still getting the old values. 
Do i need to update any variables in any other places ? can you please guide me.

Error Message is:
Name: Old sub name(Sub ID)
The running command stopped because the prefrence variables "ErrorActionPreference" or commonparamater is set to stop: Resouce group "" could not be found.

**Comment by sainathyv** [comment]
I am able to find the issue. TIA

**Reply by helderpinto** [reply, marked as answer]
Hi, @sainathyv . I am glad you were able to resolve the issue. For you awareness, when moving AOE to a different resource group, you need to update the following variables:

- `AzureOptimization_StorageSinkRG`
- `AzureOptimization_LogAnalyticsWorkspaceRG` - only if you deployed a dedicated Log Analytics workspace (i.e., you are not reusing a previously existing workspace)
